exericse4/exercise3.js

```diff
@@ -0,0 +1,4 @@
+const sum = (array) => {
+  if (array.length < 1) return 0;
+  return array.pop() + sum(array)
+};
```

Comment:
You didn't consider nested elements, in question they said **recursively** which means array might or might not be nested ones , Consider that also